api/src/test/java/io/strimzi/api/kafka/model/KafkaTest.java

```diff
@@ -80,15 +82,27 @@ public void testCertificationAuthorityBuilderAndInts() throws URISyntaxException
 
     @Test
     public void testNewListenerSerialization() throws URISyntaxException {
-        List<GenericKafkaListener> listeners = Collections.singletonList(
-                new GenericKafkaListenerBuilder()
+
+        List<GenericKafkaListener> listeners = new ArrayList<>(2);
+        listeners.add(new GenericKafkaListenerBuilder()
                         .withName("lst")
                         .withPort(9092)
                         .withType(KafkaListenerType.INTERNAL)
                         .withTls(true)
                 .build()
         );
 
+        GenericKafkaListenerConfiguration genericKafkaListenerConfiguration = new GenericKafkaListenerConfiguration();
+        genericKafkaListenerConfiguration.setCreateBootstrapService(false);
+
+        listeners.add(new GenericKafkaListenerBuilder()
+                .withName("snd")
+                .withPort(9094)
+                .withType(KafkaListenerType.LOADBALANCER)
+                .withTls(true)
+                .withConfiguration(genericKafkaListenerConfiguration)
+                .build());
```

Comment:
I guess it is fine to keep this ... but TBH it is not really needed. I do not think this option affects the serialization in any significant way which is  what is really tested here.

Reply:
Yes, it's just a boolean value, not a complex object. I take a second thought before adding them here next time